CrowsZA/functions/fn_scatterPatternOddPattern.sqf

```diff
@@ -0,0 +1,73 @@
+/*/////////////////////////////////////////////////
+Author: Crowdedlight+WindWalker
+			   
+File: fn_scatterPatternOddPattern.sqf
+Parameters: center pos in AGL, number of spots to generate, offset in meters between points, altitude
+Return: array of positions for TP each player
+
+Generates an list of 3d positions for an uniqe tp pattern. Used together with scatterTeleport
+
+*///////////////////////////////////////////////
+params ["_targetPos", "_amount", "_offset", "_targetAltitude"];
+
+// create array of _amount positions to fill with teleport positions, We don't use the first position which is the targetPos, but build around it. 
+private _tpPositions = [];
+
+// length of segment, we start with just 1, as we grow outwards
+private _segmentLength = 1;
+
+// current position (i, j) and how much of current segment we passed.
+private _i = _targetPos select 0;
+private _j = _targetPos select 1;
+private _targetAltitude = _targetAltitude;
+
+private _segmentPassed = 0;
+
+private _row = 0;
+private _tick = 0;
+
+// loop and build positions
+for "_k" from 1 to _amount do { 
+	
+	private _tmpI = 0;
+	private _tmpJ = 0;
+	
+	switch (_tick) do {
+		case 0:  { _tmpI = _i + 0  	 + 0 	  ;		_tmpJ = _j + 0 +	(10 * _row)};
```

Comment:
Doesn't have an offset, so the first unit is placed where you click, but the rest is offset in x direction and creates a figure ``_offset`` away.